specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -612,131 +681,545 @@
+    },
+    "RenewalTermDetails": {
+      "description": "Renewal term details of the product.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "billingFrequency": {
+          "description": "Billing frequency of the product under the subscription.",
+          "type": "string",
+          "readOnly": true
+        },
+        "productTypeId": {
+          "description": "The ID of the product.",
+          "type": "string",
+          "readOnly": true
+        },
+        "quantity": {
+          "description": "The number of licenses",
+          "type": "integer",
+          "format": "int64"
+        },
+        "skuId": {
+          "description": "The SKU ID of the product",
+          "type": "string",
+          "readOnly": true
+        },
+        "termDuration": {
+          "description": "The duration. Example P1Y and P1M",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Reseller": {
+      "description": "Details of the reseller.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "resellerId": {
+          "description": "The MPN ID of the reseller.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "description": "The name of the reseller.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "MoveBillingSubscriptionEligibilityErrorCode": {
+      "type": "string",
+      "description": "Error code of the transfer validation response.",
+      "enum": [
+        "AccountIsLocked",
+        "AssetNotActive",
+        "AssetHasCap",
+        "BillingAccountInactive",
+        "BillingProfilePastDue",
+        "CrossBillingAccountNotAllowed",
+        "DestinationBillingProfileNotFound",
+        "DestinationBillingProfileInactive",
+        "DestinationBillingProfilePastDue",
+        "DestinationInvoiceSectionNotFound",
+        "DestinationInvoiceSectionInactive",
+        "InvalidDestination",
+        "InvalidSource",
+        "InvoiceSectionIsRestricted",
+        "InsufficientPermissionOnDestination",
+        "InsufficientPermissionOnSource",
+        "MarketplaceNotEnabledOnDestination",
+        "ProductNotFound",
+        "ProductInactive",
+        "ProductTypeNotSupported",
+        "SourceBillingProfilePastDue",
+        "SourceInvoiceSectionInactive",
+        "SubscriptionNotActive",
+        "SubscriptionTypeNotSupported"
+      ],
+      "x-ms-enum": {
+        "name": "subscriptionTransferValidationErrorCode",
+        "modelAsString": true
+      }
+    },
+    "MoveBillingSubscriptionRequest": {
+      "type": "object",
+      "description": "Request parameters to transfer billing subscription.",
+      "properties": {
+        "destinationInvoiceSectionId": {
+          "type": "string",
+          "description": "The destination invoice section id."
+        },
+        "destinationBillingProfileId": {
```

Comment:
destinationInvoiceSectionId contains already billingProfile, so if invoice section is specified we don't need it.
Are we supporting moving by using only destinationBillingProfileId? If so, which invoice section we will use?